packages/sanity/src/core/versions/components/GlobalBundleMenu.tsx

```diff
@@ -0,0 +1,142 @@
+
+export function GlobalBundleMenu(): JSX.Element {
+  const router = useRouter()
+
+  // eslint-disable-next-line no-warning-comments
+  // FIXME REPLACE WHEN WE HAVE REAL DATA
+  const bundles = BUNDLES
+
+  // eslint-disable-next-line no-warning-comments
+  // FIXME REPLACE WHEN WE HAVE REAL DATA
+  const {currentVersion, setCurrentVersion, isDraft} = useContext(VersionContext)
+  const [createBundleDialogOpen, setCreateBundleDialogOpen] = useState(false)
+
+  const handleBundleChange = useCallback(
+    (bundle: Version) => () => {
+      const {name} = bundle
+
+      if (name === 'drafts') {
+        router.navigateStickyParam('perspective', '')
+      } else {
+        router.navigateStickyParam('perspective', `bundle.${name}`)
+      }
+
+      setCurrentVersion(bundle)
+    },
+    [router, setCurrentVersion],
+  )
+
+  const handleGoToLatest = useCallback(
+    () => () => {
+      router.navigateStickyParam('perspective', '')
+
+      setCurrentVersion(LATEST)
+    },
+    [router, setCurrentVersion],
+  )
+
+  /* create new bundle */
+
+  const handleCreateBundleClick = useCallback(() => {
+    setCreateBundleDialogOpen(true)
+  }, [])
+
+  const handleCancel = useCallback(() => {
+    setCreateBundleDialogOpen(false)
+  }, [])
+
+  const handleSubmit = useCallback(
+    () => (value: Bundle) => {
+      // eslint-disable-next-line no-console
+      console.log('create new bundle', value.name)
+
+      setCreateBundleDialogOpen(false)
+    },
+    [],
+  )
+
+  return (
+    <>
+      <MenuButton
+        button={
+          <Button mode="bleed" padding={0} radius="full">
+            <VersionIcon
+              tone={currentVersion?.tone}
+              icon={isDraft ? undefined : currentVersion?.icon}
+              openButton
+              padding={2}
+              // eslint-disable-next-line @sanity/i18n/no-attribute-string-literals
+              title={isDraft ? 'Latest' : currentVersion?.title}
+            />
+          </Button>
+        }
+        id="global-version-menu"
+        menu={
+          <Menu>
+            <MenuItem
+              iconRight={isDraft ? <CheckmarkIcon /> : undefined}
+              onClick={handleGoToLatest()}
+              pressed={false}
+              // eslint-disable-next-line @sanity/i18n/no-attribute-string-literals
+              text="Latest"
+            />
+            {bundles.length > 0 && (
+              <>
+                <MenuDivider />
+              </>
+            )}
+            {bundles
+              .filter((b) => b.name !== 'draft')
+              .map((b) => (
+                <MenuItem key={b.name} onClick={handleBundleChange(b)} padding={1} pressed={false}>
+                  <Flex>
+                    <VersionIcon tone={b.tone} icon={b.icon} padding={2} />
+
+                    <Box flex={1} padding={2} style={{minWidth: 100}}>
+                      <Text size={1} weight="medium">
+                        {b.name === 'draft' ? 'Latest' : b.title}
```

Comment:
`b.name === 'draft'` is not possible right? A filter for this is applied in [row 99](https://github.com/sanity-io/sanity/pull/7039/files#diff-0eddbc52536e7776df00ffd12d620855ccd15f8291b941d7b11eee3569c90ed2R99)

Reply:
Yep, the filter was added after this condition (and I forgot to remove these), good catch!